apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -17,4 +17,43 @@ describe('When: I use the reading list feature', () => {
       )
     );
   });
+  it('Should remove book from reading list upon clicking UNDO in snack Bar', async () => {
+    await browser.get('/');
+    await browser.wait(
+        ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
+    );
+    const form = await $('form');
+    const input = $('input[type="search"]');
+    await input.sendKeys('redux');
+    await form.submit();
+    const button = await $$('[data-add-book="add-book-to-reading-list"] [ng-reflect-disabled="false"]');
+    button[0].click();
+    const snackbarUndoButton =  await $$('snack-bar-container .mat-button');
+    await browser.wait(
+        ExpectedConditions.textToBePresentInElement(snackbarUndoButton[0], 'Undo')
+    );
+    snackbarUndoButton[0].click();
```

Comment:
Here you are clicking the button, but you are not actually checking if it actually removed the book from the reading list after you clicked "Undo". I think you should make sure that the undo action really worked!

Reply:
Missed on it. Update spec in recent commit.